app/models/actor_movie.rb

```diff
@@ -0,0 +1,4 @@
+class ActorMovie < ApplicationRecord
```

Comment:
```suggestion
class ActorsMovie < ApplicationRecord
```
Because "actors" is plural in your table name in the db, it needs to be plural here. Rails isn't smart enough to know when there are multiple words in a class name and some are plural and some are not.